cmd/hyperkube/hyperkube.go

```diff
@@ -162,7 +162,7 @@ func (hk *HyperKube) Run(args []string) error {
 		hk.Usage()
 		return err
 	}
-
+	s.Flags().SetInterspersed(false)
```

Comment:
Agreed- this is something the servers should decide for themselves. Addressed in 09e3d0913a53e342ea4dbb563cf0788d8b5a5756